load_fixtures.py

```diff
@@ -0,0 +1,17 @@
+from os import chdir, getcwd, listdir, path, system
```

Comment:
Nice work, just a small issue `chdir` is unused here

Reply:
@Tevinthuku  thanks for pointing this out. Let me remove it.